extensions/wikia/Email/EmailController.class.php

```diff
@@ -507,15 +507,13 @@ protected function getUserFromName( $username ) {
 			throw new Fatal( 'Required username has been left empty' );
 		}
 
-		// Allow an anonymous user to be specified
```

Comment:
Moved this below the test for `$username` being an object.  When `$username` is an object this test emits a warning about  being unable to convert the object into an integer.